### DISCUSSION TITLE
[Task]: Implement SimpleNet Paper

### DISCUSSION THREAD

**Discussion by willyfh** [open-edge-platform/anomalib#1607] [Feature Requests] [answered]
### What is the motivation for this task?

Implementation of SimpleNet , one of the sota anomaly detection models published in CVPR 2023.

### Describe the solution you'd like

📄 [Paper](https://openaccess.thecvf.com/content/CVPR2023/papers/Liu_SimpleNet_A_Simple_Network_for_Image_Anomaly_Detection_and_Localization_CVPR_2023_paper.pdf)
🧑‍💻 [Code](https://github.com/DonaldRR/SimpleNet)

**Comment by samet-akcay** [comment]
I agree that we should have this in anomalib. Let's convert this to a feature request

**Comment by abc-125** [comment, marked as answer]
The paper was promising, but it looks like the authors repeatedly evaluated their model on the test data during the training and then reported the best results; see this [issue](https://github.com/DonaldRR/SimpleNet/issues/7). Results without this scenario-specific tuning can be found in the v2 EfficientAD [paper](https://arxiv.org/pdf/2303.14535.pdf), and they are worse than PatchCore (which was supposedly the main inspiration for SimpleNet), especially on the VisA dataset (87.9 vs 94.3 mean image level AUROC).

**Comment by samet-akcay** [comment]
That's interesting. @jpcbertoldo proposed an approach to potentially avoid such issues, and create a more consistent benchmarks. For details you could refer to the anomalib PR #1557  and the [paper](https://arxiv.org/pdf/2401.01984.pdf).

**Comment by samet-akcay** [comment]
Due to the concerns raised for this paper, let's close this
